apiary.apib

```diff
@@ -8958,7 +8958,7 @@ Detail of Merge request also contains `content` of changed objects.
             "title": "Refactor DB extractors",
             "description": "DB exctractor have new columns now",
             "state": "in_review",
-            "content": "",
+            "content": {},
```

Comment:
toto se musi upravit u vsech odpovedi MR endpointu. Bohuzel se mi to v rychlosti nepovedlo presunout do Data structures, abychom to pouzivali pres referenci. Tak bud to zkus a uprav to pak na jednom miste, nebo to prosim prepis vsude.

Zaroven si ale myslim, ze minimalne u odpovedi [request-review](https://keboola.docs.apiary.io/#reference/development-branches/merge-requests/put-mr-in-review) by tady nemel byt `{}`, ale nejaky obsah, aby z tohoto examplu bylo patrne co to vraci . A nemusel si to konzument provolavat